Gems/LmbrCentral/Code/Source/Builders/LuaBuilder/LuaBuilderWorker.cpp

```diff
@@ -33,6 +33,20 @@ namespace LuaBuilder
 {
     namespace
     {
+// carbonated begin
+#if defined(CARBONATED)
+        static const AZStd::unordered_map<AZStd::string, AZStd::string> AssetExtensionReplacementMap =
+        {
+            {"",                 ".lua"},     // assume that a file without extension maybe a lua file
```

Comment:
Let's not make this assumption.  The AP doesn't process files without an extension as LUA files, and neither should we.  If a dev referenced a lua file without extension, then that is their error to fix.  

TL;DR:  remove the "" -> ".lua" mapping

Reply:
Removed this rule.
It allows to skip any file path without extension.